interface/ceed-basis.c

```diff
@@ -784,15 +793,19 @@ int CeedSymmetricSchurDecomposition(Ceed ceed, CeedScalar *mat, CeedScalar *lamb
     CeedScalar mu = t_nn - t_nnm1 * t_nnm1 / (d + copysign(sqrt(d * d + t_nnm1 * t_nnm1), d));
     CeedScalar x  = mat_T[p + n * p] - mu;
     CeedScalar z  = mat_T[p + n * (p + 1)];
+
     for (CeedInt k = p; k < n - q - 1; k++) {
       // Compute Givens rotation
       CeedScalar c = 1, s = 0;
+
       if (fabs(z) > tol) {
         if (fabs(z) > fabs(x)) {
-          CeedScalar tau = -x / z;
+          const CeedScalar tau = -x / z;
+
           s = 1 / sqrt(1 + tau * tau), c = s * tau;
         } else {
-          CeedScalar tau = -z / x;
+          const CeedScalar tau = -z / x;
+
           c = 1 / sqrt(1 + tau * tau), s = c * tau;
```

Comment:
Probably so the auto-formatter would put them on the same line, but I think most forms of the comma operator are confusing and unnecessary. It's mostly useful for macros, since you often want those to evaluate as an expression instead of multiple statements. The comma operator does often show up in `for` loop initialization and increments since `;` has a special meaning there.

Reply:
Yeah, I'm familiar with the comma usage in for loops, but didn't know it was applicable outside of that.

Reply:
Dropped all the instances of the comma operator I could find.